Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int[] arr = new int[]{1, 2, 3, 4, 5};
+        System.out.println("Even numbers:");
+
+        for (int i = 0; i < arr.length; i = i * 2)
+        {
+            System.out.println(arr[i]);
+        }
+    }

```

Comment:
**Code Structure Issue**: The method Even should follow Java naming conventions. Method names should start with a lowercase letter. <br> **Fix**: Rename the method from Even to even or any other name that starts with a lowercase letter, ensuring it accurately represents the method's functionality. <br> **Code Suggestion**: 
 ```
 public static void even(String[] args)
 ```